w3d_vgui.lua

```diff
@@ -80,26 +80,37 @@ end
 -- Check if the mouse is hovering something on the screen
 function w3d.IsHovered(x, y, w, h)
 
-    local iMouseX, iMouseY = w3d.tCache.iMouseX, w3d.tCache.iMouseY
-    
+    local iMouseX, iMouseY = w3d.GetMousePos()
+
     if iMouseX and iMouseY then
         return iMouseX >= x and iMouseX <= x + w and iMouseY >= y and iMouseY <= y + h
     end
 
-    return false
-
 end
 
 -- Create a new vgui element
-function w3d.Create(sType, sId, x, y, w, h, fcCallback, fcPaint)
+function w3d.Create(sType, sId, x, y, w, h, fcCallback, fcPaint, bNoButton)
```

Comment:
This parameter has nothing to do with the issue. There's actually no way to use that, why would you want to create a button that is not a button? Just draw your text into the 3D2D without using this function and it will work.